src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -53,9 +53,9 @@ object Parser {
   // cache maps a key (obtained by hasing file path and file content) to the parse result
   private var sourceCache: Map[SourceCacheKey, (Either[Vector[ParserError], PProgram], Positions)] = Map.empty
 
-  /** computes the key for caching a particular source. This takes the name as well as content into account */
-  private def getCacheKey(source: Source): SourceCacheKey = {
-    val key = source.name ++ source.content
+  /** computes the key for caching a particular source. This takes the name, the specOnly flag, and the file's content into account */
```

Comment:
```suggestion
  /** computes the key for caching a particular source. This takes the name, the specOnly flag, and the file's contents into account */
```

Reply:
I would keep it singular as a file only has one content

Reply:
Content and contents have slightly different meanings. https://getproofed.com/writing-tips/word-choice-content-vs-contents/

Reply:
Thanks, I was not aware of this. Sorry